rtos/Mutex.h

```diff
@@ -29,6 +29,8 @@
 
 #include "platform/NonCopyable.h"
 #include "platform/ScopedLock.h"
+#include "platform/mbed_toolchain.h"
+#include "platform/mbed_error.h"
```

Comment:
These should probably be in the cpp file, not here?

Reply:
Removed since cpp already includes "mbed_error.h".